clients/java/signalr/src/main/java/com/microsoft/signalr/Subscription.java

```diff
@@ -5,17 +5,23 @@
 
 import java.util.List;
 
+/**
+ * A type that represents being subscribed to invocations to a specific target.
+ */
 public class Subscription {
     private final CallbackMap handlers;
     private final InvocationHandler handler;
     private final String target;
 
-    public Subscription(CallbackMap handlers, InvocationHandler handler, String target) {
+    Subscription(CallbackMap handlers, InvocationHandler handler, String target) {
         this.handlers = handlers;
         this.handler = handler;
         this.target = target;
     }
 
+    /**
+     * Unsubscribes from a target by removing all handlers associated with it.
```

Comment:
It only unsubscribes the handler represented by this registration though right? So given:

```java
Subscription a = connection.on("foo", ...);
Subscription b = connection.on("foo", ...);
a.unsubscribe();
```

`b` is still subscribed right? The doc seems to imply this removes all handlers.

How about: "Removes the client method handler represented by this subscription" ?

Reply:
Yes, that's right. Mix this up with the remove method.